drafts/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "_comment1": "this is required only for typescript. once this is fixed https://github.com/microsoft/TypeScript/issues/33079 we can remove this hack",
+    "name": "@primer/react/drafts",
+    "types": "../lib-esm/drafts/index.d.ts",
+    "main": "../lib-esm/drafts/index.js",
+    "type": "module",
+    "sideEffects": false
+  }
+  
```

Comment:
ohhhhh, this is big brain time! 🧠

Reply:
Yeah, so without these all the imports work except the typescript build fails. This hack I found in @primer/behavior simply worked here.